src/client/delphi_epidata.py

```diff
@@ -601,8 +610,13 @@ def covidcast(
     if 'format' in kwargs:
       params['format'] = kwargs['format']
 
+    if 'cache_timeout' in kwargs:
+      cache_timeout = kwargs['cache_timeout']
+    else:
+      cache_timeout = None
```

Comment:
since there's a default value here, would just make the cache timeout variable a function argument with a default None value